RealmSwift/Object.swift

```diff
@@ -250,6 +250,16 @@ public final class DynamicObject : Object {
         }
         return nil
     }
+
+    /// :nodoc:
+    public override func valueForKey(key: String) -> AnyObject? {
```

Comment:
This breaks `realm`, `objectSchema`, `invalidated`, and 'description` (same as for the objc version)